en/docs/secure-health-apis/guides/getting-patientid.md

```diff
@@ -0,0 +1,96 @@
+# Retrieving the Patient ID
+
+In this example we will walk through the steps that how we can get the Patient ID by executing the SMART on FHIR flow in Healthcare Accelerator. 
```

Comment:
Description needs to reflect that these steps walkthrough on configuring a SMART App Launch flow for a patient user.

Reply:
Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee